src/domains/checkout-app/04_apim_checkout.tf

```diff
@@ -338,6 +338,7 @@ resource "azurerm_api_management_api_operation_policy" "transaction_authorizatio
   xml_content = templatefile("./api/checkout/checkout_ecommerce/v1/_auth_request.xml.tpl", {
     ecommerce_xpay_psps_list = var.ecommerce_xpay_psps_list
     ecommerce_vpos_psps_list = var.ecommerce_vpos_psps_list
+    ecommerce_npg_psps_list  = var.ecommerce_npg_psps_list
```

Comment:
Also here, should we need to modify only ecommerce-app domain with `ecommerce/checkout` api path?